### ISSUE TITLE
Is this still under development?

### ISSUE THREAD

**Issue by ntnsndr** [demcra/demcra#24]
I just discovered demcra and I think it's fascinating. I recently co-organized a major conference on [Platform Cooperativism](http://platformcoop.net), and I think demcra seems to fit very nicely into this movement. I would love to see more development and use. Is anybody here? What kind of support do you need?

**Comment by AlexPoulsen** [comment]
I was only ever doing some icons for it and contributing ideas. I haven't done anything with it for months now. Make sure to talk to acerix, he was doing code.

I'll consider helping again, but I'm busy with college so I won't have a ton of time.

I think that we would be wise to plan a little more before starting, I think the reason we both stopped is a lack of a clear sense of direction.
It was always democracy + Reddit, but that was mostly it on large-scale direction.

**Comment by ntnsndr** [comment]
Thanks, @AlexPoulsen. @acerix, anything to add?

I'm not a developer, but I can share this project with tech-coop developers that might be interested. I would love to see a platform like this be embraced by the emerging platform-coop community.

**Comment by AlexPoulsen** [comment]
Same, I think the web is the perfect place for democracy.
Lots of people/devices, tech to count lots of votes easily. A forum also lends itself to democracy I think. Elect moderators, vote on real world topics. Add in a sort of politwoops thing, twitter integration.
This could be where voting (real world) happens in a few years.

**Comment by ntnsndr** [comment]
Absolutely. Projects like liquid democracy and democracyOS demonstrate the viability of this for sure—and also how online platforms create new opportunities for democratic practice.

I'm thrilled to find this site, and I'm sorry I didn't find it in time to feature it at the November conference. People there would've been very excited.

**Comment by ntnsndr** [comment]
Also, is this project open source? I see that this github project is only discussion, not actual code.

**Comment by AlexPoulsen** [comment]
Yeah, I don't think acerix ever uploaded his code. I consider it open source. I may be possible to host the barebones code on Github pages and have the images/content on a publicly accessible server, since I've tried to get Github LFS (large file storage) working for a personal site and I couldn't figure it out, even after their tutorial.
I can do basic static html/css, but that's it.

**Comment by acerix** [comment]
Hi Nathan,  Thanks for sharing your interest, I agree it fits well with what you are doing.  Cool conference by the way, first I've heard of it.

The project is still alive, but has been on the backburner for the past few months, I've only done a few small fixes.  I'm hoping to get more board members so it can actually be run democratically and also I'm more motivated when others are involved.

Everything will be open source (MIT license) but I've only pushed a few source files to github so far.  I'm a bit paranoid that the code might help someone find a way to exploit the site, and potentially break into my development server where I have other projects.  For now I've been pushing one file at a time after auditing, but if there is interest in helping with the coding, it wouldn't cost much to put demcra on it's own server, I just don't want that extra expense until there is a benefit.

**Comment by ntnsndr** [comment]
Glad to hear it is still moving forward. Looks like a really important project, and I'd love to see it succeed.
